core/global/platform.gd

```diff
@@ -178,7 +178,12 @@ func _read_dmi() -> PLATFORM:
 		return PLATFORM.AOKZOE_GEN2
 
 	# ASUS
-	elif product_name.contains("ROG Ally RC71L") and vendor_name == "ASUSTeK COMPUTER INC.":
+	elif product_name in [
```

Comment:
I was testing on the Ally X and needed it to work. I can move these to a separate PR if desired.